docs/status.md

```diff
@@ -135,7 +135,7 @@ one of the following tracking labels.
 | TanhOp |yes||||no|
 | TorchIndexSelectOp |no||||no|
 | TraceOp |no||||no|
-| TransposeOp |no||||no|
+| TransposeOp |yes||||yes|
```

Comment:
Let's also check whether TransposeOp's verifier and shape function conform to the newly written spec. If they do, let's flip `yes*` in the table to `yes`. If they don't let's open tickets about this.

Reply:
done.